echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/github/GithubService.java

```diff
@@ -16,20 +16,26 @@
 
 package com.netflix.spinnaker.echo.github;
 
-import retrofit.client.Response;
-import retrofit.http.*;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.http.Body;
+import retrofit2.http.GET;
+import retrofit2.http.Header;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
 
 public interface GithubService {
   @POST("/repos/{repo}/statuses/{sha}")
-  Response updateCheck(
+  Call<Response<ResponseBody>> updateCheck(
       @Header("Authorization") String token,
-      @Path(value = "repo", encode = false) String repo,
+      @Path(value = "repo") String repo,
```

Comment:
How come `encode = false` goes away?

Reply:
Aaah, the Path annotation in retrofit2 uses encoded, not encode, but it defaults to false....where encode defaults to true in retrofit1.  I think this means we'd add `encoded = true` to every @Path annotation that doesn't specify encode to avoid a behavior change....except that feels painful.